GLTF/src/GLTFAsset.cpp

```diff
@@ -440,6 +440,205 @@ GLTF::BufferView* packAccessorsForTargetByteStride(std::vector<GLTF::Accessor*>
 	return bufferView;
 }
 
+#ifdef USE_DRACO
+bool GLTF::Asset::compressPrimitives() {
+	int totalPrimitives = 0;
+	for (GLTF::Primitive* primitive : getAllPrimitives()) {
+		totalPrimitives++;
+		auto draco_ext_itr = primitive->extensions.find("KHR_draco_mesh_compression");
+		if (draco_ext_itr == primitive->extensions.end()) {
+			// No extension exists.
+			std::cout << "No extension exists for the primitive.\n";
```

Comment:
Add a verbose option to switch these on/off or remove them if they aren't really necessary.